shopping/src/pages/Main.js

```diff
@@ -0,0 +1,29 @@
+import React, {useEffect, useState} from "react";
```

Comment:
```suggestion
import {useEffect, useState} from "react";
```

React default import 부분은 생략해도 되는 것으로 알고 있어요.